entity-framework/core/extensions/index.md

```diff
@@ -359,3 +359,11 @@ It also has many useful features commonly used especially on web development. Fo
 
 [GitHub repository](https://github.com/SitholeWB/Pagination.EntityFrameworkCore.Extensions)
 [NuGet](https://www.nuget.org/packages/Pagination.EntityFrameworkCore.Extensions)
+
+### Laraue.EfCoreTriggers
+Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite.
```

Comment:
Please specify the supported EF Core versions (like it is done for all other extensions in this list)